net/shared/src/main/scala/rapture/net/http.scala

```diff
@@ -182,7 +182,7 @@ object HttpSupport {
             }
         }
 
-        new HttpResponse(mapAsScalaMap(conn.getHeaderFields()).toMap.mapValues(_.to[List]), statusCode, is)
+        new HttpResponse(mapAsScalaMapConverter(conn.getHeaderFields()).asScala.toMap.mapValues(x => collectionAsScalaIterableConverter(x).asScala.toList), statusCode, is)
```

Comment:
Minor style point: I prefer `foo.bar { x => ... }` rather than `foo.bar(x => ...)`, but prefer `(_.baz)` for underscore-based lambdas!